libs/dal/src/repositories/change/change.entity.ts

```diff
@@ -12,6 +12,8 @@ export class ChangeEntity {
 
   _environmentId: EnvironmentId;
 
+  user: Pick<UserEntity, '_id' | 'firstName' | 'lastName' | 'profilePicture'>;
```

Comment:
Should we add this one here only for the ChangeRepository.getList populate?
Maybe instead we can create a response type with it for getList, wdyt?

Reply:
I agree - I will create type for this particular method